src/pages/singleArticle/tests/__snapshots__/editArticle.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Edit Article Page Component Should render without crashing 1`] = `ReactWrapper {}`;
```

Comment:
@felixkiryowa this is not what a snapshot is supposed to look like. You can use react-test-renderer